test_results.py

```diff
@@ -0,0 +1,172 @@
+import unittest
+import sys
+import os
+
+# Add the parent directory to the Python path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from utils.custom_tsv_parser import ParseTSVFile
+
+
+def get_scode_cscode_id(data):
+    # Create an id with scode and center with sorted values of scode and center
+    school_centers = []
+    for row in data:
+        scode = int(row["scode"])
+        center = int(row["cscode"])
+        sccode_center_id = sorted((scode, center))
+        sccode_center_id = "_".join(map(str, sccode_center_id))
+        school_centers.append(sccode_center_id)
+    return school_centers
+
+
+class TestSchoolCenter(unittest.TestCase):
+    """_Tests to validate the outcome of the output are matching
+    as per the requirements
+
+    Needs the ouptut from the results/school-center.tsv and
+    results/school-center-distance.tsv files to be present
+
+    """
+
+    def setUp(self):
+        self.school_center_file = "results/school-center.tsv"
+        self.school_center_distace_file = "results/school-center-distance.tsv"
+        self.school_center_pref_file = "sample_data/prefs.tsv"
+
+    def test_scode_student_count_not_more_than_200(self):
```

Comment:
एक विद्यालयको परिक्षार्थी संख्या हेरी **सकभर** १००, २०० भन्दा बढी परीक्षार्थी एकै केन्द्रमा नपर्ने गरी बाँढ्न पर्ने 
200 is not a hard limit. implementing #13 will fail this test.

Reply:
OK will add it only as a warning for now

Reply:
Done